src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs

```diff
@@ -0,0 +1,1912 @@
+    /// <since_tizen> 6 </since_tizen>
+    public enum CommandType
+    {
+        /// <summary>
+        /// foreground command type
+        /// </summary>
+        Foreground = 1,
+        /// <summary>
+        /// background command type
+        /// </summary>
+        Background = 2,
+        /// <summary>
+        /// widget command type
+        /// </summary>
+        Widget = 3,
+        /// <summary>
+        /// system command type
+        /// </summary>
+        System = 4,
+        /// <summary>
+        /// system background command type
+        /// </summary>
+        SystemBackground = 5,
+        /// <summary>
+        /// exclusive command type
+        /// </summary>
+        Exclusive = 6,
+        /// <summary>
+        /// Undefined command
+        /// </summary>
+        Undefined = -1
+    };
+
+    /// <summary>
+    /// Enumerations of recognition mode
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public enum RecognitionModeType
+    {
+        /// <summary>
+        /// Default mode
+        /// </summary>
+        StopBySilence,
+        /// <summary>
+        /// Restart recognition after rejected result
+        /// </summary>
+        RestartAfterReject,
+        /// <summary>
+        /// Continuously restart recognition - not support yet
+        /// </summary>
+        RestartContinuously,
+        /// <summary>
+        /// Start and stop manually without silence
+        /// </summary>
+        Manual,
+        /// <summary>
+        /// Undefined
+        /// </summary>
+        Undefined = -1
+    }
+
+    /// <summary>
+    /// Enumerations of send event type
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public enum SendEventType
+    {
+        /// <summary>
+        /// Send text event to vc engine
+        /// </summary>
+        Text,
+        /// <summary>
+        /// Send list event to vc engine
+        /// </summary>
+        ListEvent,
+        /// <summary>
+        /// Send haptic event to vc engine
+        /// </summary>
+        HapticEvent,
+    }
+
+    /// <summary>
+    /// Enumerations for audio channels
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public enum AudioChanelType
+    {
+        /// <summary>
+        /// 1 channel, mono
+        /// </summary>
+        Mono = 0,
+        /// <summary>
+        /// 2 channels, stereo
+        /// </summary>
+        Stereo
+    }
+
+    /// <summary>
+    /// Enumerations of audio types
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public enum AudioType
+    {
+        /// <summary>
+        /// Signed 16bit audio type, Little endian
+        /// </summary>
+        PcmS16Le = 0,
+        /// <summary>
+        /// Unsigned 8bit audio type
+        /// </summary>
+        PcmU8
+    }
+
+    /// <summary>
+    /// Enumeration for TTS feedback events
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public enum FeedbackEventType
```

Comment:
How about `FeedbackType`?
Why do you need `Event` word?

Reply:
Removed 'Event' word.